File: python/accounts/balance.py
import json
from squid_py.ocean.ocean import Ocean
from ..config import CONFIG_FILE

ocean = Ocean(CONFIG_FILE)
balance = ocean.main_account.get_balance()
response = {
    'eth': balance.eth,
    'ocn': balance.ocn,
}
print("__result__{}".format(json.dumps(response, indent=2, sort_keys=True)))

File: node/accounts/balance.js
const { Ocean, Logger } = require('@oceanprotocol/squid')
let config = require('../config');

(async () => {
    Logger.log('running accounts/balance with config', config)
    const ocean = await Ocean.getInstance(config)
    const accounts = await ocean.getAccounts()
    Logger.log(accounts)
    const balance = await Promise.all(accounts.map(async account => {
        return {
            'id': account.id,
            'balance': await account.getBalance()
        }
    }))

    Logger.log(`__result__${JSON.stringify(balance, null, 2)}`)
})()
